src/controllers/accounts/accounts.date.controller.ts

```diff
@@ -61,7 +62,13 @@ export const render = async (req: Request, res: Response, next: NextFunction): P
     await sessionService.setReasonInContextAsString(req.chSession, req.query.reasonId as string);
   }
   let dateStr;
-  const reason: ReasonWeb = await reasonService.getCurrentReason(req.chSession) as ReasonWeb;
+  let reason: ReasonWeb;
+  try {
+    reason = await reasonService.getCurrentReason(req.chSession) as ReasonWeb;
+  } catch (err) {
+    logger.info("Error caught retrieving current reason for accounts date")
```

Comment:
worth seeing if the the logger version we have can use logger.infoRequest instead (where we have access to the request) and pass req in if available (i think it logs a request id then to help match user journeys in the logs)

Reply:
I'm afraid not, it looks like the logger isn't using a CH logger at all. It's just built on top of log4js. Probably does need an update but I don't want to introduce too many changes in this change

Reply:
that's fine, just wanted to check.